test/src/System/Vector3.cpp

```diff
@@ -158,4 +159,50 @@ TEST_CASE("sf::Vector3 class template", "[system]")
             CHECK_FALSE(firstEqualVector != secondEqualVector);
         }
     }
+
+    SECTION("Structured bindings")
+    {
+        sf::Vector3i vector(1, 2, 3);
+
+        SECTION("destructure by value")
+        {
+            auto [x, y, z] = vector;
+
+            CHECK(x == 1);
+            CHECK(y == 2);
+            CHECK(z == 3);
+
+            static_assert(std::is_same_v<decltype(x), decltype(vector.x)>);
+
+            x = 3;
+
+            CHECK(x == 3);
+            CHECK(vector.x == 1);
+        }
+
+        SECTION("destructure by ref")
+        {
+            auto& [x, y, z] = vector;
+
+            CHECK(x == 1);
+            CHECK(y == 2);
+            CHECK(z == 3);
+
+            static_assert(std::is_same_v<decltype(x), decltype(vector.x)>);
+
+            x = 3;
+
+            CHECK(x == 3);
+            CHECK(vector.x == 3);
+        }
+    }
+
+    SECTION("Constexpr support")
```

Comment:
Would it make sense to add a tiny test for the `constexpr` conversion constructor (e.g. `float` -> `int`)?

Reply:
We already have a `"Conversion constructor"` section, the fact that it is now `constexpr` doesn't affect its behavior at all.